src/fragments/lib/logging/common/remote_configuration/remote_configuration.mdx

```diff
@@ -0,0 +1,220 @@
+The AWSCloudWatchLoggingPlugin log levels can be configured remotely.  When configured, the Amplify Logger will fetch the remote log levels, defined in a `remoteloggingconstraints.json` file and deployed to an S3 bucket, via an API Gateway endpoint that is integrated with a lambda.  The log levels defined in the remote logging constraints file will overwrite the local log levels and will persist across app sessions.
+
+You will need to provision and maintain your own backend resources.  The following provides a CDK construct example to provision the required AWS resources.
+
+<Callout warning>
```

Comment:
I feel like this callout isn't achieving much here - do we need it really?

Reply:
Callout removed.